index.html

```diff
@@ -67,18 +82,18 @@ <h2>Why?</h2>
         </ul>
       </section>
 
-
+      <hr>
       <section>
         <blockquote>
-          Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.
+          "Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt."
         </blockquote>
         <p>
-          Mx. Ipsum
+          -- Mx. Ipsum
```

Comment:
I think there is a way to add a "--" in css instead of changing the html directly! I would try using the pseudo-class ":before" with "content: '--'"